### PR TITLE
feat(oakprevlessontitles): create oak prev lesson title

### PR DESCRIPTION
**Pull request by benprotheroe**
# How to review this PR

_Leave this text block for the reviewer_

- Check [component hierarchy](https://miro.com/app/board/uXjVNnKBgyk=/?share_link_id=59445593794) is followed correctly
- Check the design [Heuristics](https://components.thenational.academy/?path=/docs/docs-howtodesigncomponents--docs#heuristics-for-component-design) have been followed
- Check [naming conventions](https://components.thenational.academy/?path=/docs/docs-namingconventions--docs) have been applied
- Check for these gotchyas:
  - Missing exports for Oak components
  - Accidental export of Internal components
  - Snapshots of unexpected components have been modified
  - Circular dependencies
  - Code duplication (via not using base components)
  - Non-functional storybook for this or related components
  - Sensitve files changed eg. atoms, or style tokens
  - Relative imports
  - Default exports

# Add your PR description below

create OakPreviousLessonHeader

## Link to the design doc

https://www.figma.com/file/zqpeecgzMGKRli2gUdJOLp/Pupil-Browse?type=design&node-id=4426%3A57791&mode=dev

## A link to the component in the deployment preview

/?path=/docs/components-organisms-pupil-oakpreviouslessonsheading--docs

## Testing instructions

go to above story on storybook, check acs

### Acceptance criteria

- [x]  Displays previous lessons with number of lessons
- [x]  Displays bulleted programme factors
- [x]  documentation where relevant

### CODE REVIEW

File: src/components/organisms/pupil/OakPreviousLessonsHeading/OakPreviousLessonsHeading.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import React from "react";
+
+import { OakBulletList } from "@/components/molecules";
+import { OakFlex, OakHeading } from "@/components/atoms";
+
+export type OakPreviousLessonsHeadingProps = {
+  numberOfLessons: number;
+  yearDescription: string;
```
Comment (kimon-satan) [inline src/components/organisms/pupil/OakPreviousLessonsHeading/OakPreviousLessonsHeading.tsx:8]: I wouldn't try to align these with Schema data types. I'd rather just go for year, examboard, subject etc. It's more confusing this way around

File: src/components/organisms/pupil/OakPreviousLessonsHeading/OakPreviousLessonsHeading.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  subjectTitle: string;
+  tierDescription?: string;
+  examBoardDescription?: string;
+};
+
+/**
+ * This component is the header for the pupil journey;
```
Comment (kimon-satan) [inline src/components/organisms/pupil/OakPreviousLessonsHeading/OakPreviousLessonsHeading.tsx:15]: this is an old comment. Can you update it pls

**Review by kimon-satan — CHANGES_REQUESTED** [review]

File: src/components/organisms/pupil/OakPreviousLessonsHeading/OakPreviousLessonsHeading.tsx (hunk 1 above)
Comment (benprotheroe) [reply src/components/organisms/pupil/OakPreviousLessonsHeading/OakPreviousLessonsHeading.tsx:8]: yep

**Review by kimon-satan — APPROVED** [review]
Can't see the changes a la github's strange "feature" but I assume you implemented them

**Comment by benprotheroe** [comment]
> @benprotheroe as Mike mentioned -
Screenshot 2024-05-07 at 15 04 56
 alignment for mobile is to the right rather than below and label colour is not implemented

i think i have made these changes; is this what youre referring to?

**Review by messmike — APPROVED** [review]
200w

**Review by RBrownsett — APPROVED** [review]

**Pull request merged**